packages/cli/src/tools/config/__tests__/index-test.ts

```diff
@@ -51,13 +51,13 @@ const removeString = (config, str) =>
     ),
   );
 
-beforeEach(() => {
-  cleanup(DIR);
+beforeEach(async () => {
+  await cleanup(DIR);
   jest.resetModules();
   jest.clearAllMocks();
 });
 
-afterEach(() => cleanup(DIR));
```

Comment:
can you rebase to latest master? we've recently changed these tests, because I started to have consistent fails here, resulting in files not being properly cleaned on time. as a mitigation, I changed the DIR value for every test case

Reply:
I've rebased with master but don't see any changes in here 🤔

Reply:
Oh my, it's not on master yet 😅 https://github.com/react-native-community/cli/pull/1151
No issue then :)